docs/docker-compose-and-github-actions.rst

```diff
@@ -0,0 +1,191 @@
+
+        services:
+          venueless:
+            image: eventyay/eventyay-video:${TAG}-latest
+            container_name: venueless
+            ports:
+              - "8375:80"
+            environment:
+              - DJANGO_SETTINGS_MODULE=venueless.settings
+              - LC_ALL=C.UTF-8
+              - IPYTHONDIR=/data/.ipython
+              - VENUELESS_COMMIT_SHA=${COMMIT}
+              - VENUELESS_DB_TYPE=postgresql
+              - VENUELESS_DB_NAME=${POSTGRES_DB}
+              - VENUELESS_DB_USER=${POSTGRES_USER}
+              - VENUELESS_DB_PASS=${POSTGRES_PASSWD}
+              - VENUELESS_DB_HOST=venueless-db
+              - VENUELESS_REDIS_URLS=redis://venueless-redis:6379/0,redis://venueless-redis1:6379/0
+              - VENUELESS_DATA_DIR=/data
+              - VENUELESS_MEDIA_URL=http://localhost:8375/media/
+              - VENUELESS_REDIS_USE_PUBSUB=true
+            depends_on:
+              - venueless-db
+              - venueless-redis
+              - venueless-redis1
+            volumes:
+              - /etc/venueless:/etc/venueless
+              - /data:/data
+            entrypoint: ["/usr/local/bin/venueless"]
+            command: ["all"]
+          venueless-webapp:
+            image: eventyay/eventyay-video:${TAG}-latest
+            container_name: venueless-webapp
+            ports:
+              - "8002:8880"
+            environment:
+              - NODE_OPTIONS=--openssl-legacy-provider
+            entrypoint: [""]
+            command: ["npm", "start", "--", "--host", "0.0.0.0"]
+            working_dir: /venueless/webapp
+
+          venueless-redis:
+            image: redis:latest
+            container_name: venueless-redis
+
+          venueless-redis1:
+            image: redis:latest
+            container_name: venueless-redis1
+
+          venueless-db:
+            image: postgres:15
+            container_name: venueless-db
+            volumes:
+              - postgres_data:/var/lib/postgresql/data/
+            environment:
+              - POSTGRES_INITDB_ARGS=--auth-host=md5
+              - POSTGRES_HOST_AUTH_METHOD=md5
+              - POSTGRES_DB=${POSTGRES_DB}
+              - POSTGRES_USER=${POSTGRES_USER}
+              - POSTGRES_PASSWORD=${POSTGRES_PASSWD}
+
+        volumes:
+          postgres_data:
+          appdata:
+
+Usage
+--------
+Ensure that the `TAG`, `POSTGRES_DB`, `POSTGRES_USER`, and `POSTGRES_PASSWD` environment variables are set before running the Docker Compose commands. These variables are typically set in the deployment script or CI/CD pipeline.
+
+GitHub Actions Workflow
+------------------------
+
+Overview
+--------
+This GitHub Actions workflow is configured to build and push Docker images to Docker Hub and deploy them to a remote server. The workflow is triggered on commits to the `development` and `master` branches and can also be manually triggered.
+
+Workflow File (`main.yml`)
+
+            name: Main workflow
+            on:
```

Comment:
This workflow is not on the repo and that's how we want it, can we remove this part of doc then?